bootstrap/bootstrap.py

```diff
@@ -0,0 +1,119 @@
+import os
+import sys
+import argparse
+from git import Repo
+
+
+class Helper:
+
+    def __init__(self, project_directory, project_name):
+        self._project_directory = project_directory
+        self._project_name = project_name
+        self._git_repo = "https://github.com/microsoft/MLOpsPython.git"
+
+    @property
+    def project_directory(self):
+        return self._project_directory
+
+    @property
+    def project_name(self):
+        return self._project_name
+
+    @property
+    def git_repo(self):
+        return self._git_repo
+
+    def clonerepo(self):
+        # Download MLOpsPython repo from git
+        Repo.clone_from(
+            self._git_repo, self._project_directory, branch="master", depth=1)
+        print(self._project_directory)
+
+    def renamefiles(self):
+        # Rename all files starting with diabetes_regression with project name
+        strtoreplace = "diabetes_regression"
+        dirs = [".pipelines", r"ml_service\pipelines"]
+        for dir in dirs:
+            dirpath = os.path.join(self._project_directory, dir)
+            for filename in os.listdir(dirpath):
+                if(filename.find(strtoreplace) != -1):
+                    src = os.path.join(self._project_directory, dir, filename)
+                    dst = os.path.join(self._project_directory,
+                                       dir, filename.replace(strtoreplace, self._project_name, 1))  # NOQA: E501
+                    os.rename(src, dst)
+
+    def renamedir(self):
+        # Rename any directory with diabetes_regression with project name
+        dirs = ["diabetes_regression"]
+        for dir in dirs:
+            src = os.path.join(self._project_directory, dir)
+            dst = os.path.join(self._project_directory, self._project_name)
+            os.rename(src, dst)
+
+    def deletedir(self):
+        # Delete unwanted directories
+        dirs = [".git", "docs", r"diabetes_regression\training\R"]
+        for dir in dirs:
+            os.system(
+                'rmdir /S /Q "{}"'.format(os.path.join(self._project_directory, dir)))  # NOQA: E501
+
+    def replaceimport(self):
+        # Replace imports with new project name
+        dirs = [r"diabetes_regression\training\test_train.py",
```

Comment:
>diabetes_regression [](start = 18, length = 19)

Any reason not to simply blindly replace the "diabetes_regression" string in all files? It's a nice unique string that shouldn't have any false-positive matches.